resources/lang/pt-br/beatmaps.php

```diff
@@ -21,108 +21,133 @@
         'update' => [
-            'error' => 'Atualizar votos falhou',
+            'error' => 'Falha ao atualizar votos',
         ],
     ],
 
     'discussions' => [
+        'edit' => 'editar',
+        'edited' => 'Editado pela última vez por :editor :update_time',
+        'message_placeholder' => 'Digite aqui para publicar',
+        'message_type_select' => 'Selecione o tipo de comentário',
+        'reply_placeholder' => 'Digite a sua resposta aqui',
+        'require-login' => 'Por favor, inicie a sessão para publicar ou responder',
+        'resolved' => 'Resolvido',
+        'title' => 'Discussões',
+
         'collapse' => [
-            'all-collapse' => 'Comprimir todos',
-            'all-expand' => 'Expandir todos',
+            'all-collapse' => 'Recolher todas',
+            'all-expand' => 'Expandir todas',
         ],
 
-        'edit' => 'edit',
-        'edited' => 'Editado pela última vez por :editor :update_time',
         'empty' => [
-            'empty' => 'Sem discussões aqui por enquanto!',
-            'filtered' => 'Nenhuma discussão encontrada com este filtro.',
+            'empty' => 'Nenhuma discussão ainda!',
+            'hidden' => 'Nenhuma discussão corresponde ao filtro selecionado.',
         ],
 
         'message_hint' => [
-            'in_general' => 'Este post vai para a discussão de geral de beatmaps. Para moddar este beatmap, comece sua mensagem com um timestamp (ex. 00:12:345).',
-            'in_timeline' => 'Para moddar vários timestamps, poste várias vezes (um post por timestamp).',
+            'in_general' => 'Esta publicação irá para discussão geral de beatmaps. Para modificar este beatmap, inicie a mensagem com o timestamp (ex: 00:12:345).',
+            'in_timeline' => 'Para modificar vários timestamps, publique várias vezes (uma publicação por timestamp).',
         ],
 
-        'message_placeholder' => 'Digite aqui para postar',
-
         'message_type' => [
-            'praise' => 'Aplaudir',
+            'praise' => 'Elogio',
             'problem' => 'Problema',
             'suggestion' => 'Sugestão',
         ],
 
-        'message_type_select' => 'Escolha Tipo do Comentário',
-
         'mode' => [
             'general' => 'Geral',
-            'timeline' => 'Linha do Tempo',
+            'timeline' => 'Linha do tempo',
         ],
 
-        'require-login' => 'Faça login para postar ou responder',
-        'resolved' => 'Resolvido',
+        'new' => [
+            'timestamp' => 'Timestamp',
+            'title' => 'Nova discussão',
+        ],
 
         'show' => [
-            'title' => 'Discussão de Beatmaps',
+            'title' => ':title mapeado por :mapper',
         ],
 
         'stats' => [
             'mine' => 'Meu',
-            'pending' => 'Pendentes',
-            'praises' => 'Aplausos',
-            'resolved' => 'Resolvidos',
+            'pending' => 'Pendente',
+            'praises' => 'Elogios',
+            'resolved' => 'Resolvido',
```

Comment:
they all should be on plural since it's going to show a list of pending, praises and resolved comments.